### ISSUE TITLE
Can't get car model, but was working on previous update (ee13f27)

### ISSUE THREAD

**Issue by aleblack** [flobz/psa_car_controller#121]
As the title says, after the last 'git pull' I now receive the error: 

`WARNING :: Can't get car model, please report an issue on github with your car model and first ten letter of your VIN`

This was not present with the previous update. BTW I have an e-208 and and (of course) the first digits of the VIN are VR3UHZKXZ.

**Comment by flobz** [comment]
Can you delete car.json and retry ?

**Comment by aleblack** [comment]
Done, but same error. 
The newly created cars.json it's exactly the same as the old  one.

**Comment by flobz** [comment]
Can you give me your car json file

**Comment by aleblack** [comment]
sure, here it is, I've redacted part of VIN and vehicle_id.

```json
[
    {
        "abrp_name": "peugeot:e208:20:50",
        "battery_power": 46,
        "brand": "Peugeot",
        "fuel_capacity": 0,
        "label": "e-208",
        "max_elec_consumption": 70,
        "max_fuel_consumption": 0,
        "vehicle_id": "xxxxxxxx",
        "vin": "VR3UHZKXZLxxxxxxx"
    }
]
```

**Comment by moulagaufre** [comment]
Yep, same here, was working yesterday, and after a git pull:
2021-05-13 13:56:48,451 :: WARNING :: Can't get car model, please report an issue on github with your car model and first ten letter of your VIN

my car 5008

**Comment by diegodanado** [comment]
same here with corsa-e with the last update

**Comment by flobz** [comment]
I updated the program to add the VIN to the error message : 4f017a255d5e99ae3c228661dea28a71f2db0273.
Can you retry with the last version,  and tell if you vin is in the message ?

**Comment by diegodanado** [comment]
on the new version same error with corsa-e VIN: VXKUHZKXZL....
and vin is not in log
2021-05-14 17:37:27,538 :: WARNING :: Can't get car model, please report an issue on github with your car model and first ten letter of your VIN : vin

**Comment by flobz** [comment]
fixed 8ccb46a !

**Comment by aleblack** [comment]
Thanks for the fix @flobz, now it's working.
